### PR TITLE
Issue 34/seo

### PR DESCRIPTION
**Pull request by harcod**
Initial attempt at META tags for SEO.  Open Graph (OG) meta for Facebook & Twitter.

image

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -9,10 +9,30 @@
     />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Make an appointment to get your COVID vaccine"
-    />
+    <meta name="keywords" content="massachusetts, mass, ma, covid, covid-19, vaccine, vaccines, vax, search, sign up, zipcode">
```
Comment (j256) [inline public/index.html:12]: All of these need a self close, right?   <meta ... />

**Review by livgust — CHANGES_REQUESTED** [review]
Hey @harcod, could you add the close `/` for all the meta tags? thanks!

File: public/index.html (hunk 1 above)
Comment (orzechowskid) [reply public/index.html:12]: according to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta), `<meta>` is a "void element" which means no children and no end tag are allowed.  so `<meta>` with no closing `/>` is correct.

Comment (harcod) [reply public/index.html]: I added closing tags for all meta

**Pull request merged**